.travis.yml

```diff
@@ -35,6 +35,7 @@ script:
   - node scripts/transpileMd.js src/app/app-welcome/app-welcome.content.md src/app/app-welcome/app-welcome.content.html
   - ng lint
   - ng test --watch false
+  - ./scripts/release.sh
```

Comment:
@AlbaLopez there's one more thing you need to do: Bash scripts can't be executed as you just create them, because it's not marked as an "executable" file for the bash. You can turn it into an executable by running `chmod +x scripts/createSentryRelease.sh`. You only have to run it once on your computer, and these changes will be recorded in git (although not visible).


https://travis-ci.org/DD-DeCaF/caffeine/builds/399181673?utm_source=github_status&utm_medium=notification